Misc/NEWS.d/next/Library/2023-07-20-06-00-35.gh-issue-106739.W1hygr.rst

```diff
@@ -0,0 +1 @@
+Adds the `rtype_cache` `set` to the `warnings.warn` message (as an addition to the type of leaked objects and the number of leaked objects already included in the `warnings.warn` message) to make debugging leaked objects easier when the `resource_tracker` module (`Lib/multiprocessing/resource_tracker.py`) finds leaked objects. Including the names of the leaked objects will help more quickly identify what was leaked and/or why the leaked object was not properly cleaned up.
```

Comment:
```suggestion
Add the ``rtype_cache`` to the warning message (as an addition to the type of leaked objects and the number of leaked objects already included in the message) to make debugging leaked objects easier when the multiprocessing resource tracker process finds leaked objects at shutdown. This helps more quickly identify what was leaked and/or why the leaked object was not properly cleaned up.
```

Reply:
I updated the entry in commit 16ec55739400259a6b02b9a4553d1217d1fc56d1. Thanks for catching the incorrect formatting for `rtype_cache`! (I used markdown formatting and missed the double backticks)